src/main/java/org/spongepowered/api/util/Color.java

```diff
@@ -387,65 +366,42 @@ public Color mixWithDyes(DyeColor... dyeColors) {
         return mixColors(newColorArray);
     }
 
-    @Override
-    public int getContentVersion() {
-        return 1;
-    }
-
-    @Override
-    public DataContainer toContainer() {
-        return DataContainer.createNew()
-                .set(Queries.CONTENT_VERSION, this.getContentVersion())
-                .set(Queries.COLOR_RED, this.getRed())
-                .set(Queries.COLOR_GREEN, this.getGreen())
-                .set(Queries.COLOR_BLUE, this.getBlue());
-    }
-
-    @Override
-    public int hashCode() {
-        return this.rgb;
-    }
-
-    @Override
-    public boolean equals(Object obj) {
-        return this == obj || obj != null && getClass() == obj.getClass() && this.rgb == ((Color) obj).rgb;
-    }
+    interface Builder extends ResettableBuilder<Color, Builder> {
```

Comment:
Individual red, blue, green setters?

Reply:
I did that in one attempt, but there isn't really a point to it. I'd say about 99% of use cases are going to be through the default methods, and it's both easier and faster to set them all at the same time. This also ensures that there's never a partial color in the builder, which will be 5x as important if we decide to ignore alpha values rather than removing them (which means we need to save a rgb value as well that would need to be updated for each of those method calls). It's an overhead that I don't think is worth it in the slightest even though it's a bit against the builder mentality - but so is the entire class anyways.